skopt/space/space.py

```diff
@@ -240,8 +240,8 @@ def __eq__(self, other):
                 self.transform_ == other.transform_)
 
     def __repr__(self):
-        return "Real(low={}, high={}, prior='{}', transform='{}')".format(
-            self.low, self.high, self.prior, self.transform_)
+        return "Real(low={}, high={}, prior='{}', transform='{}', name='{}')".format(
+            self.low, self.high, self.prior, self.transform_, self.name)
```

Comment:
Can you also do that for dimensions of other types, for consistency? (Integer, etc).

Reply:
Disclaimer: this is not a nitpick.

Reply:
Also, since names are `None` by default, I am not sure we actually want to print `name=None` in the default case. What do others think?

Reply:
Now there's a good review request! :-)

It is already done in PR #579 which also fixes other things with `Dimension`-names. I just needed a single one for testing in this PR.

Reply:
Can you backport from #579 please? (That's why we prefer self-contained atomic PRs that target a specific goal, to avoid these situations...)

Reply:
I'm not an expert at git so I thought this would merge seamlessly as the change is identical for the two PRs. But this code was actually not required in this PR, I only needed it for some print-testing. So I just removed it from this PR.